docs/spl/atom/atom-guide-1a-configure.markdown

```diff
@@ -12,90 +12,84 @@ next:
   title: Importing your code into Atom
 ---
 
-
-
-Download the Streams plugins for Atom
+Download the Streams Packages for Atom
 --------------------------------------------------
 
-If you haven't already done so, [download and install
-Atom](https://atom.io).
-
-*atom-ide-ui, ide-ibmstreams, language-ibmstreams-spl,
-build-ibmstreams*:
-
--   Go to **Atom** \> **Preferences** \> **Install** \> **Packages**
-    (might be different for Windows)
+If necessary, [download and install Atom](https://atom.io). To install the Streams Packages for Atom:
 
--   Search for each of the above packages and install it:
+1. In Atom, open the package installation screen:
+* Mac: **Atom > Preferences > Install > Packages**
+* Windows&reg;: **File > Settings > Install > Packages**
+  ![Screenshot of Atom package install](/streamsx.documentation/images/atom/jpg/install-package.jpg)
 
-    ![Screenshot of Atom package install](/streamsx.documentation/images/atom/jpg/install-package.jpg)
+2. Search for each of the following packages and install them:
+* atom-ide-ui
+* ide-ibmstreams
+* language-ibmstreams-spl
+* build-ibmstreams
 
+3. In Atom, open the Streams themes:
+* Mac: **Atom > Preferences > Install > Themes**
+* Windows&reg;: **File > Settings > Install > Themes**
 
--   Install the Streams themes:
-
-    -   **Atom** \> **Preferences** \> **Install** \> **Themes**
-
-    -   Search for and install either of *streams-dark-syntax* or
-        *streams-light-syntax*.
+4. Search for and install the following themes if you want to:
+* streams-dark-syntax
+* streams-light-syntax
 
 Create an instance of the Streaming Analytics service
 ---------------------
 
-Instead of downloading the Streams compiler and runtime to create your applications, you will use the Streaming Analytics service, a cloud based version of Streams. Applications created in Atom are sent to the Streaming Analytics to be compiled and executed.
-
-
-* If you have a Streaming Analytics service in [IBM Cloud](https://console.ng.bluemix.net/), make sure that it is started and running.
+Instead of downloading the tools to set up your own Streams environment, you can use the Streaming Analytics cloud service. Applications that are created in Atom can be sent to the Streaming Analytics service to be compiled and run.
 
-* To create a new Streaming Analytics service:
-  1. Visit to the [IBM Cloud web portal](https://www.ibm.com/cloud-computing/bluemix/) and sign in (or sign up for a free account).
-
-  2. Go to the [Streaming Analytics service](https://console.bluemix.net/catalog/services/streaming-analytics) page within the Catalog.
-
-  3. Enter the service name and then click **Create** to set up your service. The service dashboard opens and your service starts automatically. The service name appears as the title of the service dashboard.
+If you already have an instance of Streaming Analytics service in [IBM Cloud](https://console.ng.bluemix.net/), make sure that it is started and running.
```

Comment:
Add a section about ICP4D as well, based on https://github.com/IBMStreams/build-ibmstreams#ibm-cloud-private-for-data-streams-add-on-instance